aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1421,8 +1421,8 @@ class QConvInt8ForBC final {
 };
 
 TORCH_LIBRARY_IMPL(quantized, QuantizedCPU, m) {
-  m.impl(TORCH_SELECTIVE_NAME("quantized::conv1d"),          QConv1dInt8<false>::run);
-  m.impl(TORCH_SELECTIVE_NAME("quantized::conv1d_relu"),     QConv1dInt8<true>::run);
+  m.impl(TORCH_SELECTIVE_NAME("quantized::conv1d.new"),          QConv1dInt8<false>::run);
+  m.impl(TORCH_SELECTIVE_NAME("quantized::conv1d_relu.new"),     QConv1dInt8<true>::run);
```

Comment:
@jerryzh168 I changed the cpu one to be `.new` to be consistent with conv2d. any objections?

Reply:
oh, I see, so cpu does not have this `new` variant before?

Reply:
in that case let's just use the old one and add a comment in GPU registrations, we don't want to break bc